src/config.ts

```diff
@@ -104,20 +104,22 @@ type FishServer = {
 	port:string;
 	aliases:string[];
 	name:string;
+  requiredTrusted?:boolean;
```

Comment:
Change this to a string representing the Perm, and resolve it at usage location
Also, create a new perm called "hardcoreServer", and set it to trusted OR paying member